kolibri/core/webpack/hooks.py

```diff
@@ -156,19 +154,24 @@ def webpack_bundle_data(self):
         }
 
     @property
+    def module_path(self):
+        return '.'.join(self.__module__.split('.')[:-1])
+
+    @property
     def build_path(self):
         """
         An auto-generated path to where the build-time files are stored,
         containing information about the built bundles.
         """
-        return resource_filename('kolibri.core', 'build')
+        return resource_filename(self.module_path, 'build')
+
+    @property
+    def static_dir(self):
```

Comment:
Oh nice :) So it can be overloaded if someone wants it differently... great default behavior!